test/utils/logger/printers/test_level.py

```diff
@@ -0,0 +1,60 @@
+import re
+import pytest
+
+from colorama import Fore, Style
+
+from magda.utils.logger.printers import LevelPrinter
+from magda.utils.logger.config import LoggerConfig
+from magda.utils.logger.parts import LoggerParts
+
+
+class TestLevelPrinter:
+    COLOR_REGEXP = r'\x1b'
+
+    @pytest.mark.parametrize(
+        "level,expected_color",
+        [
+            (LoggerConfig.Level.INFO, Fore.WHITE),
+            (LoggerConfig.Level.WARNING, Fore.YELLOW),
+            (LoggerConfig.Level.ERROR, Fore.RED),
+            (LoggerConfig.Level.DEBUG, Fore.GREEN),
+            (LoggerConfig.Level.CRITICAL, Fore.RED)
+        ]
+    )
+    def test_should_print_correctly_any_level(self, level, expected_color):
```

Comment:
Just naming, but we would like to print correctly each level - not just any of them:
```suggestion
    def test_should_print_correctly_level(self, level, expected_color):
```